service/src/main/java/uk/gov/hmcts/reform/fpl/service/orders/generator/UploadedOrderDocumentGenerator.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.reform.fpl.service.orders.generator;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.fpl.enums.OrderStatus;
+import uk.gov.hmcts.reform.fpl.enums.YesNo;
+import uk.gov.hmcts.reform.fpl.enums.docmosis.RenderFormat;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.common.DocumentReference;
+import uk.gov.hmcts.reform.fpl.service.DocumentDownloadService;
+import uk.gov.hmcts.reform.fpl.service.DocumentSealingService;
+import uk.gov.hmcts.reform.fpl.service.docmosis.DocumentConversionService;
+
+@Component
+@RequiredArgsConstructor(onConstructor_ = {@Autowired})
+public class UploadedOrderDocumentGenerator {
+
+    private final DocumentSealingService documentSealingService;
+    private final DocumentConversionService documentConversionService;
+    private final DocumentDownloadService documentDownloadService;
+
+    public OrderDocumentGeneratorResult generate(
+        CaseData caseData,
```

Comment:
minor but can probs be on the same as the decalartion line

Reply:
👍